src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -488,6 +485,21 @@ private void handleUnresolvedTypeProblem(ExecutionContext context, ExecutionStra
 
     }
 
+    /**
+     * Called to complete a null value.
+     *
+     * @param executionContext contains the top level execution parameters
+     * @param parameters       contains the parameters holding the fields to be executed and source object
+     *
+     * @return a {@link FieldValueInfo}
+     *
+     * @throws NonNullableFieldWasNullException if a non null field resolves to a null value
+     */
+    private FieldValueInfo getFieldValueInfoForNull(ExecutionContext executionContext, ExecutionStrategyParameters parameters) {
+        CompletableFuture<ExecutionResult> fieldValue = completeValueForNull(executionContext, parameters);
+        return FieldValueInfo.newFieldValueInfo(NULL).fieldValue(fieldValue).build();
+    }
```

Comment:
I had a look at this in the complete code.  I can see that the value as not being completed into a CF and hence this would intere with the completion of data loaders and so on.